links/src/commonTest/kotlin/com/splendo/kaluga/links/manager/MockLinksManager.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.links.manager
+
+class MockLinksManagerBuilder : LinksManager.Builder {
+    override fun create(onLinksStateChange: LinksStateChange): LinksManager =
+        DefaultLinksManager(onLinksStateChange, MockLinksValidator())
+}
+
+class MockLinksValidator : LinksValidator {
+    override fun isValid(url: String): Boolean {
+        return url == "valid"
+    }
+}
+
+object TestConstants {
+    val VALID_URLS = listOf(
+        "https://test.io",
+        "http://test.io",
+        "https://test.io?isValid=true",
+        "https://test.io?list_1=first&list_2=second&list_3=third"
```

Comment:
Do they valid without `/` before `?`

Reply:
what do you mean?

Reply:
I mean `test.io/?foo=bar` is valid, but I doubt about `test.io?foo=bar`

Reply:
I guess it pass because the browser can resolve it by appending that `/`